app/webpacker/stylesheets/_definition-list.scss

```diff
@@ -0,0 +1,53 @@
+%govuk-list--description {
```

Comment:
Is this ported from somewhere?
If it's homemade can we use a different prefix to `govuk-` as that implies it's in the design system.

On GOV.UK we used `app-` a lot.

Reply:
Agreed! Will tackle in one hit when this work is done